pkg/apis/core/register.go

```diff
@@ -97,6 +98,7 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&ConfigMap{},
 		&ConfigMapList{},
 	)
+	scheme.AddKnownTypes(SchemeGroupVersion, &auth.TokenRequest{})
```

Comment:
I am sorry that I missed your review because of other things.

I want to ask clearly, is this because this file is not the best location for "" "v1" TokenRequest? So we need to migrate the "" "v1" TokenRequest to other schemes. For example, audit.k8s.io scheme, authentication.k8s.io scheme.

Reply:
Uh, I haven’t updated the code yet, I don’t know if I can request a review.

Reply:
I think this pr should modify the func below to use correct subresource gvr. https://github.com/kubernetes/kubernetes/blob/efd7df5c9993a326189da356c0466629b7683a77/staging/src/k8s.io/apiserver/pkg/audit/request.go#L114